contracts/contracts/RaidenMicroTransferChannels.sol

```diff
@@ -42,8 +42,8 @@ contract RaidenMicroTransferChannels {
     }
 
     struct ClosingRequest {
-        uint32 settle_block_number;
         uint192 closing_balance;
+        uint32 settle_block_number;
```

Comment:
@loredanacirstea Curious: Why change the order of the struct members here? Alphabetical?

Reply:
Just for consistency with the above (sizes). Also mentioned as a code style improvement in https://github.com/raiden-network/microraiden/issues/134#issuecomment-347144899 and I said why not ..